windows-installer/generate_sdl_agent_exe.ps1

```diff
@@ -135,18 +144,15 @@ $env:Path = [System.Environment]::GetEnvironmentVariable("Path","Machine") + ";"
 #  STEP 4 - INSTALL THE GEMS
 ##############################
 #
-# Install the needed gems for google fleuntd to works.
+# Install the needed gems for google fluentd to works.
 #
 # These are all a known set of versions which work together. Be sure things work if you
 # update any of them.
-#
-# We install all gems with '--no-document' so we don't pull in
-# unneeded docs that bloat the file size (and also seem to cause issues with unzipping).
 ###############################
 
-& $GEM_CMD install fluentd:1.4.2 --no-document
```

Comment:
The Linux build does not install `fluentd` via the `gem` command, but rather has omnibus build it from source (see [config](https://github.com/GoogleCloudPlatform/google-fluentd/blob/master/config/software/fluentd.rb)). Since we don't use omnibus on Windows, should we not add it to `plugin_gems.rb` windows section? If not, where do you install the `fluentd` gem?

Reply:
I can add it, if you think it's important. This is the list of gems that are getting installed as it is now, including fluentd. It probably gets installed as a dependency for something else:
addressable (2.7.0)
aws-eventstream (1.0.3)
aws-partitions (1.149.0)
aws-sdk-core (3.48.3)
aws-sdk-kms (1.16.0)
aws-sdk-s3 (1.36.0)
aws-sdk-sqs (1.13.0)
aws-sigv4 (1.1.0)
bigdecimal (default: 1.3.4)
certstore_c (0.1.6)
cmath (default: 1.0.0)
concurrent-ruby (1.1.5)
cool.io (1.5.4 x86-mingw32)
csv (default: 1.0.0)
date (default: 1.0.0)
dbm (default: 1.0.0)
declarative (0.0.10)
declarative-option (0.1.0)
did_you_mean (1.2.0)
dig_rb (1.0.1)
domain_name (0.5.20190701)
etc (default: 1.0.0)
faraday (0.17.0)
fcntl (default: 1.0.0)
ffi (1.11.1 x86-mingw32)
ffi-compiler (1.0.1)
ffi-win32-extensions (1.0.3)
fiddle (default: 1.0.0)
fileutils (default: 1.0.2)
fluent-config-regexp-type (1.0.0)
fluent-plugin-detect-exceptions (0.0.12)
fluent-plugin-google-cloud (0.7.23)
fluent-plugin-kubernetes_metadata_filter (2.4.0)
fluent-plugin-multi-format-parser (1.0.0)
fluent-plugin-prometheus (1.4.0)
fluent-plugin-record-modifier (2.0.1)
fluent-plugin-record-reformer (0.9.1)
fluent-plugin-rewrite-tag-filter (2.2.0)
fluent-plugin-s3 (1.1.10)
fluent-plugin-systemd (1.0.2)
fluent-plugin-webhdfs (1.2.3)
fluentd (1.7.4 x86-mingw32, 1.6.3 x86-mingw32)
gdbm (default: 2.0.0)
google-api-client (0.30.8)
google-cloud-core (1.4.1)
google-cloud-env (1.3.0)
google-cloud-logging (1.6.6)
google-cloud-monitoring (0.29.5)
google-cloud-trace (0.36.0)
google-gax (1.8.1, 1.7.1)
google-protobuf (3.9.0 x86-mingw32)
googleapis-common-protos (1.3.9)
googleapis-common-protos-types (1.0.4)
googleauth (0.9.0)
grpc (1.24.0 x86-mingw32, 1.22.0 x86-mingw32)
http (4.2.0)
http-accept (1.7.0)
http-cookie (1.0.3)
http-form_data (2.1.1)
http-parser (1.2.1)
http_parser.rb (0.6.0)
httpclient (2.8.3)
io-console (default: 0.4.6)
ipaddr (default: 1.2.0)
jmespath (1.4.0)
json (2.2.0, default: 2.1.0)
jwt (2.2.1)
kubeclient (4.5.0)
lru_redux (1.1.0)
memoist (0.16.0)
mime-types (3.3)
mime-types-data (3.2019.1009)
mini_mime (1.0.2)
minitest (5.10.3)
msgpack (1.3.1 x86-mingw32)
multi_json (1.14.1)
multipart-post (2.1.1)
net-telnet (0.1.1)
netrc (0.11.0)
opencensus (0.5.0)
opencensus-stackdriver (0.3.0)
openssl (default: 2.1.2)
os (1.0.1)
power_assert (1.1.1)
prometheus-client (0.9.0)
psych (default: 3.0.2)
public_suffix (4.0.1)
quantile (0.2.1)
rake (12.3.0)
rdoc (default: 6.0.1)
recursive-open-struct (1.1.0)
representable (3.0.4)
rest-client (2.1.0 x86-mingw32)
retriable (3.1.2)
rly (0.2.3)
scanf (default: 1.0.0)
sdbm (default: 1.0.0)
serverengine (2.1.1 x86-mingw32)
sigdump (0.2.4)
signet (0.12.0)
stackdriver-core (1.4.0)
stringio (default: 0.0.1)
strptime (0.2.3)
strscan (default: 1.0.0)
systemd-journal (1.3.3)
test-unit (3.2.7)
thread_safe (0.3.6)
tzinfo (2.0.0, 1.2.5)
tzinfo-data (1.2019.3)
uber (0.1.0)
unf (0.1.4)
unf_ext (0.0.7.6)
uuidtools (2.1.5)
webhdfs (0.8.0)
webrick (default: 1.4.2)
win32-api (1.8.0 universal-mingw32)
win32-event (0.6.3)
win32-ipc (0.6.6)
win32-service (0.8.10)
windows-api (0.4.4)
windows-pr (1.2.6)
xmlrpc (0.3.0)
yajl-ruby (1.4.1)
zlib (default: 1.0.0)